### DISCUSSION TITLE
Problème ngrok status session denied

### DISCUSSION THREAD

**Discussion by thomass251** [community/community#108156] [Discussions]
### Select Topic Area

Question

### Body

Bonjour,

   je n'arrive pas à me connecter à ngrok, il me met status session denid et j'ai activé le hotspot, j'ai redemarré plusieurs fois, j'ai cherché toute la journée, auriez vous une solution, je débute un peu dans le codage(termux, python, ruby.
Pourriez vous m'aider, s'il vous plait ....Je ne suis pas sur que c'est le bon endroit pour poster le message mais s'il vous plait, ne me virez pas car je perds espoir....
Merci
Cordialement

**Comment by queenofcorgis** [comment]
Hi @thomass251 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing locking any posts in languages other than English for now, including this one.

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this ngrok like their Slack community: https://ngrok.com/slack. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by thomass251** [comment]
Bonjour @queenofcorgis, I can't seem to get to ngrok with termux, it tells me access status session denied, so I allow myself to ask for help.... Could you help me because I've been looking all day and I'm starting to lose hope, I'm just starting out so I don't know much about it...

**Reply by queenofcorgis** [reply]
Bonjour @thomass251, the slack link I included above or one of [these areas ](https://wiki.termux.com/wiki/Community)for Termux would be your best avenues for assistance as the community and my expertise are with GitHub, I haven't used Termux or ngrok in all honesty.

**Comment by thomass251** [comment]
Hello @reinedecorgis
, I would like to use absolutely ngrok .... I'd like to find the solution and I don't give up easily, and I really need it...
